### PR TITLE
docs: change from `anyhow` to `color-eyre`

### PR DESCRIPTION
**Pull request by zacharyftw**
Change `anyhow` to `color_eyre` in all tutorials 
Issue : https://github.com/ratatui-org/website/issues/192

NOTE : *This is a Work-In-Progress PR*

### CODE REVIEW

**Comment by orhun** [comment]
Hello!

> NOTE : This is a Work-In-Progress PR

Feel free to mark this as "draft" until it is ready for review 🐻

**Comment by zacharyftw** [comment]
@orhun it's done !

**Review by orhun — APPROVED** [review]

File: code/how-to-collapse-borders/src/bin/problem.rs
Comment (kdheepak) [inline code/how-to-collapse-borders/src/bin/problem.rs]: Is this necessary? My understanding was we only needed `color_eyre::install()` if it want to set up panic hooks.

I'd remove it from this because we have other sections on how to set up panic hooks that work well with terminal user interfaces, and installing it twice can cause errors (e.g. if a user copy pastes this code but already has a panic hook set up).

Comment (zacharyftw) [reply code/how-to-collapse-borders/src/bin/problem.rs]: @kdheepak I noticed no build errors were popping up, so I'll take it out soon.

**Review by orhun — APPROVED** [review]

**Comment by kdheepak** [comment]
This looks good to me and I don't think it conflicts with @joshka's changes, so I'm good to merge after @joshka gives it an approval! 

Thanks for making the PR!

**Review by joshka — APPROVED** [review]

**Pull request merged**
